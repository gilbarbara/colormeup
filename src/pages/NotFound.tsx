import { FlexCenter, NonIdealState } from '@gilbarbara/components';

export default function NotFound() {
  return (
    <FlexCenter data-component-name="NotFound" flex>
      <NonIdealState icon="paintbrush" type="not-found" />
    </FlexCenter>
  );
}
