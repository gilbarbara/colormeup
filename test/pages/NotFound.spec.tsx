import { render, screen } from '@testing-library/react';

import NotFound from '~/pages/NotFound';

describe('NotFound', () => {
  render(<NotFound />);

  it('should render properly', () => {
    expect(screen.getByTestId('NotFound')).toMatchSnapshot();
  });
});
