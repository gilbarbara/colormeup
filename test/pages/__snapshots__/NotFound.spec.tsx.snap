// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound > should render properly 1`] = `
<div
  class="css-6nmq0b"
  data-component-name="NotFound"
>
  <div
    class="css-91o5wy"
    data-component-name="NonIdealState"
  >
    <div
      class="css-z5ifs4"
      data-component-name="Flex"
    >
      <span
        class="css-1l1reyq"
        data-component-name="Icon"
        name="paintbrush"
      />
    </div>
    <div
      class="css-q34js2"
      data-component-name="Box"
    >
      <h2
        class="css-knznkd"
        data-component-name="H2"
      >
        Page not found
      </h2>
      <p
        class="css-1g18taf"
        data-component-name="Paragraph"
      >
        <span
          class="css-onif9v"
          data-component-name="Text"
        >
          We are sorry, but the page you requested doesn't exist.
        </span>
      </p>
    </div>
  </div>
</div>
`;
